### DISCUSSION TITLE
Adding Prisma

### DISCUSSION THREAD

**Discussion by jonwalz** [jonwalz/crypto-data#9] [General]
I think we should use Prisma for the database on this project. It seems pretty easy to setup and maintain. 

https://www.prisma.io/

Starting this discussion as a way to chat about it.

**Comment by jonwalz** [comment]
Start here https://www.prisma.io/docs/getting-started/setup-prisma/add-to-existing-project/relational-databases-typescript-postgres

**Reply by jonwalz** [reply]
We might need to create a database on Heroku https://www.heroku.com/postgres

**Reply by jonwalz** [reply]
Found more info here https://www.prisma.io/dataguide/postgresql/5-ways-to-host-postgresql#databases-managed-by-cloud-providers

**Reply by jonwalz** [reply]
Or this apparently we can host it on their platform? https://www.prisma.io/dataplatform

**Comment by jonfreihofer** [comment]
Seems like if we go with Prisma, it would be less headache to just use their all inclusive platform
